CDP4SiteDirectory.Tests/TeamComposition/TeamCompositionDisconnectViewModelTestFixture.cs

```diff
@@ -0,0 +1,102 @@
+﻿// -------------------------------------------------------------------------------------------------
+// <copyright file="TeamCompositionBrowserViewModelTestFixture.cs" company="RHEA System S.A.">
+//   Copyright (c) 2015 RHEA System S.A.
```

Comment:
I was told to set the year to 2015-2019 for files that you change.
Same for headers that contain the names of developers that worked on the file. You are allowed to add your name to those.
Don't let this comment hold you from merging by the way...